src/commands/community/nft/ticker.ts

```diff
@@ -122,12 +128,12 @@ async function viewTickerInfo(
 
 function composeTickerSelectionResponse(
   suggestions: ResponseCollectionSuggestions[],
-  symbol: string,
+  query: string,
   msg: Message
 ) {
   const opt = (s: ResponseCollectionSuggestions): MessageSelectOptionData => ({
     label: `${s.name} (${s.symbol})`,
-    value: `${s.address}_${s.chain}_${msg.author.id}`,
+    value: `${query}_${s.name}_${s.symbol}_${s.address}_${s.chain}_${s.chain_id}_${msg.author.id}`,
```

Comment:
please make sure this does not exceed maximum length allowed by discord (if any)